### DISCUSSION TITLE
Special case for binding to exactly one workload execution cluster?

### DISCUSSION THREAD

**Discussion by MikeSpreitzer** [kubestellar/kubestellar#703] [Ideas]
While an EdgePlacement generally binds to an arbitrary number of workload execution clusters, there is a significant body of usage in which that number is intended to be exactly 1.  In this case there is no summarization problem; the source object's status section is properly designed to handle this case and arguably the reported state should propagate back to there.

**Comment by clubanderson** [comment]
@MikeSpreitzer I am turning this into an issue to be resolved for PoC2023q3
